_cxx/envoy.mk

```diff
@@ -23,6 +23,9 @@ RSYNC_EXTRAS ?=
   ENVOY_DOCKER_VERSION ?= $(BASE_ENVOY_RELVER).$(ENVOY_COMMIT).$(ENVOY_COMPILATION_MODE)
   ENVOY_DOCKER_TAG ?= $(ENVOY_DOCKER_REPO):envoy-$(ENVOY_DOCKER_VERSION)
   ENVOY_FULL_DOCKER_TAG ?= $(ENVOY_DOCKER_REPO):envoy-full-$(ENVOY_DOCKER_VERSION)
+
+  # Set to non-empty to enable compiling Envoy in FIPS mode.
+  FIPS_MODE ?=
```

Comment:
I'm pretty sure that you also must include `$(FIPS_MODE)` in `ENVOY_DOCKER_VERSION` for things to work correctly:

```
ENVOY_DOCKER_VERSION ?= $(BASE_ENVOY_RELVER).$(ENVOY_COMMIT).$(ENVOY_COMPILATION_MODE).$(if $(FIPS_MODE),FIPS,noFIPS)
```

or the like. I'll defer to @LukeShu here though.

Reply:
Thanks for the suggestion @kflynn!